HeinOnline.js

```diff
@@ -2,14 +2,14 @@
 	"translatorID": "3dcbb947-f7e3-4bbd-a4e5-717f3701d624",
 	"label": "HeinOnline",
 	"creator": "Frank Bennett",
-	"target": "^https?://(www\\.)?heinonline\\.org/HOL/(?:LuceneSearch|Page|IFLPMetaData)\\?",
+	"target": "^https?://(www\.)?heinonline\.org/HOL/(?:LuceneSearch|Page|IFLPMetaData)\\?",
```

Comment:
Undo this change.

The point in a regexp refers to any character and therefore you have to escape it here `\.` in scaffold which will result in double escaped `\\.` here.